alpine_test.go

```diff
@@ -105,11 +105,11 @@ func TestAlpine(t *testing.T) {
 	}
 
 	assert.Panics(t, func() {
-		attribute("foo", "bar", "baz")
+		g.Attr("foo", "bar", "baz")
```

Comment:
I tried using `g.Attr(...)`. The problem is that it scapes some characters and Alpine expects valid JS expressions. That's why I wrote the `attribute` function.

Did you run the tests with `g.Attr(...)` again? Maybe I missed something.

Reply:
just discovered it myself and did some testing:
`g.Attr` always escapes, but the browser engines convert them just fine, so no gremlins there.

Reply:
just looked on the [tests of the htmx lib](https://github.com/maragudk/gomponents-htmx/blob/main/htmx_test.go), author is testing `g.Node` against `g.Node` instead of the final output. makes more sense tbh

Reply:
rewrote the tests, also added a Makefile to make test easier and create a cover file

Reply:
Thank you very much for your work. I've merged your pull request.